### ISSUE TITLE
What could be causing a gap between the bottom of the scrollView and the top of the inputBar?

### ISSUE THREAD

**Issue by Eric-4Privacy** [MessageKit/MessageKit#1112]
What could be causing this gap between the bottom of the ScrollView and the top of the inputBar? As soon as I begin editing, the issue resolves itself.

After loading initial messages before pressing the inputTextView:
![IMG_1229](https://user-images.githubusercontent.com/40239606/60197121-ceea4580-980c-11e9-9b38-5fd4d23cad84.PNG)

After pressing the inputTextView: (There is a new system message bc I took a screenshot but notice that the gap is gone)
![IMG_1230](https://user-images.githubusercontent.com/40239606/60197129-d27dcc80-980c-11e9-8583-7d68cf447d60.PNG)

After dismissing the keyboard: (There is another new system message bc I took another screenshot but notice the gap is gone)
![IMG_1231](https://user-images.githubusercontent.com/40239606/60197132-d4e02680-980c-11e9-82a6-495e618e5dac.PNG)

  - iOS 12.3.1
  - Swift 5
  - MarqueeLabel/Swift (3.2.1)
  - MessageInputBar/Core (0.4.1)
  - MessageKit (2.0.0):
    - MessageInputBar/Core

**Comment by marcetcheverry** [comment]
Try examining the view hierarchy at runtime with Xcode to see which one causes it.

**Comment by Eric-4Privacy** [comment]
The problem was solved by editing MessagesViewController+Keyboard.swift in the following way:

original:
`internal func requiredInitialScrollViewBottomInset() -> CGFloat {
        guard let inputAccessoryView = inputAccessoryView else { return 0 }
        return max(0, inputAccessoryView.frame.height + additionalBottomInset - automaticallyAddedBottomInset)
}`

After edit:
`    internal func requiredInitialScrollViewBottomInset() -> CGFloat {
        guard let inputAccessoryView = inputAccessoryView else { return 0 }
        return max(0, additionalBottomInset - automaticallyAddedBottomInset)
}`

I took out the piece about inputAccessoryView.frame.height. From what I can see, the collectionView.Bottom is equal to the inputAccessoryView.Top so I don't know why the height of the inputAccessoryView is being accounted for in the contentInset of the CollectionView.

**Comment by Eric-4Privacy** [comment]
UPDATE: 
Turns out the problem was actually caused from setting `tabBar.isTranslucent = false` on my tabBar controller. It was causing some issues elsewhere so I set `tabBar.isTranslucent` back to true and I noticed my previous edit to `requiredInitialScrollViewBottomInset` needed to be reversed. My iPhone 6s now works perfectly. My iPhone 10 still has a very small initial gap but it is so small that it is hardly noticeable so I am not going to investigate why for now.
